templates/meta-sharing.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
We'll need to remember to take this out before merging (and add it in via jetpack-helper instead).